BiasAndFairnessServers/src/routers/bias_and_fairness.py

```
@@ -0,0 +1,17 @@
+
+@router.post("/upload")
+async def upload_model(
+    model: UploadFile = Form(...),
+    data: UploadFile = Form(...),
+    target_column: str = Form(...), 
+    sensitive_column: str = Form(...), 
+):
+    return await handle_upload_controller(model, data, target_column, sensitive_column)
+
+@router.get("/metrics/{id}")
+async def get_metrics(id: int):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)